README.md

```diff
@@ -1,77 +1,42 @@
 # Automated Security Response on AWS
 
-[🚀 Solution Landing
-Page](https://aws.amazon.com/solutions/implementations/automated-security-response-on-aws/)
-\| [🚧 Feature
+[🚀 Solution Landing Page](https://aws.amazon.com/solutions/implementations/automated-security-response-on-aws/) \| [🚧
+Feature
 request](https://github.com/aws-solutions/aws-security-hub-automated-response-and-remediation/issues/new?assignees=&labels=feature-request%2C+enhancement&template=feature_request.md&title=)
 \| [🐛 Bug
 Report](https://github.com/aws-solutions/aws-security-hub-automated-response-and-remediation%3E/issues/new?assignees=&labels=bug%2C+triage&template=bug_report.md&title=)
 
-Note: If you want to use the solution without building from source, navigate to
-Solution Landing Page
-
-## Table of contents
```

Comment:
My understanding is that Table of contents is part of the standard readme template. Why remove it?

Reply:
Can you point me to that template on slack? I've never seen it.

I removed it because GitHub generates a ToC automatically from headings. The one in the document is redundant and has to be kept up to date manually.

<img width="925" alt="Screenshot 2023-03-14 at 08 14 47" src="https://user-images.githubusercontent.com/99816433/224998150-031fae36-d3cc-416f-906d-470a4551d5dc.png">